### PR TITLE
ADD autoprefixer

### PR DESCRIPTION
**Pull request by serapath**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ module.exports = function (opts, cb) {
 
   function complete (err, src) {
     if (opts.transform && !err) src = opts.transform(src)
-
+    if (opts.autoprefixer) {
+      var autoprefixer = require('autoprefixer-core');
```
Comment (joeybaker) [inline index.js]: Can we move this require to the top of the file?

**Comment by joeybaker** [comment]
Just the one note about moving the require. Thanks for the patch!

**Comment by serapath** [comment]
done

**Comment by joeybaker** [comment]
Published as 3.3.0

**Pull request merged**